Assets/ComethSDK/Scripts/Adapters/ConnectAdaptor.cs

```diff
@@ -42,16 +44,17 @@ public async Task Connect(string burnerAddress = "")
 		{
 			if (!string.IsNullOrEmpty(burnerAddress))
 			{
-				await VerifyWalletAddress(burnerAddress);
-				_signer = await BurnerWalletService.GetSigner(burnerAddress, _api,
-					Constants.GetNetworkByChainID(ChainId).RPCUrl);
+				_signer = await EoaFallbackService.GetSigner(_api, _provider, burnerAddress, encryptionSalt);
```

Comment:
Just before this call I would have put the MigrationLogic to move v1 private key to, the encrypted model.